projects/origin-release/rhel-7-golang-1.13/Dockerfile

```diff
@@ -6,7 +6,7 @@
 #
 FROM ocp/builder-base:rhel-7-golang-1.13
 
-ENV VERSION=1.13.5 \
+ENV VERSION=1.13.7 \
```

Comment:
I am confused. I though the rhel one is just a placeholder for ART that needs to consume golang RPM for rhel. They are stuck with 1.13.4 no matter what's set here. We shouldn't make bumps that get us out of sync. We just had several ART builds broken by that last week.

Reply:
Good to know; thanks for the heads up.